Remotion/Data/DomainObjects.UnitTests/IntegrationTests/StorageClassTransactionTest.cs

```diff
@@ -59,7 +57,8 @@ public void Commit_Rollback_ExistingObject ()
       computer.DateTimeTransactionProperty = referenceDateTime;
       computer.EmployeeTransactionProperty = referenceEmployee;
 
-      CheckValueAfterCommitAndRollback(computer, referenceDateTime, referenceEmployee);
+      using (var _ = DatabaseAgent.OpenNoDatabaseWriteSection())
```

Comment:
Sorry, styleguide: using is always done with braces. 

BTW: using as block vs using as statement: I'm okay with the statement when it's a resource that doesn't require a defined lifetime (e.g. a resourcemanager stream result, database reader etc), but always when the lifetime is important (scopes, probably datebase transactions/connections). Don't know if that's somehting we want to write analyzer for.

The braces, those we could enforce via analyzer (https://re-motion.atlassian.net/browse/RMTEAM-446)